### PR TITLE
feat(#54): package-contains-multiple-parts lint

### LINKED ISSUE #54: the `+package` meta may have more than one part, which is wrong
Currently, this is possible:

```
+package hello world
```

This is wrong, since only one (and exactly one) part is expected.

### PR DESCRIPTION
**Pull request by h1alexbel**
In this pull I've introduced new lint: `package-contains-multiple-parts` that issues `critical` defect if more than one part is used in `+package` tail. `incorrect-package` lint now checks only the first part of the `+package` tail, instead of full one.

closes #54
History:
- **feat(#54): check the first part**
- **feat(#54): package-contains-multiple-parts**
- **feat(#54): motive**

### CODE REVIEW

File: src/main/resources/org/eolang/lints/critical/package-contains-multiple-parts.xsl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="package-contains-multiple-parts" version="2.0">
+  <xsl:output encoding="UTF-8" method="xml"/>
+  <xsl:template match="/">
+    <defects>
+      <xsl:for-each select="/program/metas/meta">
+        <xsl:variable name="meta-head" select="head"/>
+        <xsl:variable name="meta-tail" select="tail"/>
+        <xsl:if test="$meta-head='package' and count(tokenize($meta-tail, ' ')) != 1">
```
Comment (yegor256) [inline src/main/resources/org/eolang/lints/critical/package-contains-multiple-parts.xsl:32]: @h1alexbel maybe you can just count the number of `part` elements here?

**Comment by h1alexbel** [comment]
@yegor256 updated pull request. Take a look again, please

File: src/main/resources/org/eolang/lints/critical/package-contains-multiple-parts.xsl [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <xsl:variable name="meta-tail" select="tail"/>
+        <xsl:if test="$meta-head='package' and count(part) != 1">
+          <xsl:element name="defect">
+            <xsl:attribute name="line">
+              <xsl:value-of select="if (@line) then @line else '0'"/>
+            </xsl:attribute>
+            <xsl:attribute name="severity">
+              <xsl:text>critical</xsl:text>
+            </xsl:attribute>
+            <xsl:text>Package meta must have the only one part "</xsl:text>
```
Comment (yegor256) [inline src/main/resources/org/eolang/lints/critical/package-contains-multiple-parts.xsl:40]: @h1alexbel maybe we can rephrase it to:

```
The +package meta may have only one part, 
which is the name of the package, 
while currently there are X parts
```

Comment (h1alexbel) [reply src/main/resources/org/eolang/lints/critical/package-contains-multiple-parts.xsl]: @yegor256 sounds good

**Comment by h1alexbel** [comment]
@yegor256 updated message

File: src/main/resources/org/eolang/lints/critical/package-contains-multiple-parts.xsl [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <xsl:variable name="parts" select="count(part)" />
+        <xsl:if test="$meta-head='package' and $parts != 1">
+          <xsl:element name="defect">
+            <xsl:attribute name="line">
+              <xsl:value-of select="if (@line) then @line else '0'"/>
+            </xsl:attribute>
+            <xsl:attribute name="severity">
+              <xsl:text>critical</xsl:text>
+            </xsl:attribute>
+            <xsl:value-of select="concat('The +package meta may have only one part, which is the name of the package, while currently there are ', $parts, ' parts')"/>
```
Comment (yegor256) [inline src/main/resources/org/eolang/lints/critical/package-contains-multiple-parts.xsl:41]: @h1alexbel why not using the formatting mechanism that we use everywhere else? This `concat` looks way less readable, in my opinion.

**Comment by h1alexbel** [comment]
@yegor256 removed `concat()` usage

**Comment by yegor256** [comment]
@h1alexbel thanks!
